### PR TITLE
Add --print-url for `edgedb ui` to only show URL

### PR DESCRIPTION
**Pull request by fantix**
Also unhide the `ui` command in `--help`.

### CODE REVIEW

**Review by 1st1 — APPROVED** [review]
I also want @elprans to look at this before merge in case he wants to consider naming the option differently

**Comment by elprans** [comment]
This is fine, but maybe we should just always print the URL? This is what URL-opening CLI commands normally do in case the browser doesn't open.

File: src/commands/ui.rs [hunk 1]
```diff
@@ -38,11 +38,15 @@ pub fn show_ui(options: &Options) -> anyhow::Result<()> {
             }
         }
     }
-    if open::that(&url).is_ok() {
+    if args.url {
+        print::echo!(url);
```
Comment (tailhook) [inline src/commands/ui.rs:42]: It should go to stdout. At least in `--url` case. So you can do:
```
url=$(edgedb ui --url)
```

**Review by tailhook — CHANGES_REQUESTED** [review]

**Comment by elprans** [comment]
> That's the thing -- i don't want the browser to open, i just want to see the url and open it myself in the browser i want

Sure, it's a separate issue.  Always printing the URL is for cases where opening the default browser didn't work for some reason so at least you can click on the link and don't have to go looking for the `--url` option.

**Comment by 1st1** [comment]
Ok, let's make it always print the URL.

File: src/commands/ui.rs [hunk 2]
```diff
@@ -38,12 +38,16 @@ pub fn show_ui(options: &Options) -> anyhow::Result<()> {
             }
         }
     }
-    if open::that(&url).is_ok() {
-        Ok(())
-    } else {
+    if !args.url && !open::that(&url).is_ok() {
```
Comment (tailhook) [inline src/commands/ui.rs:41]: Why we don't print error returned by `open::that` ? Sometimes users may want to see why we couldn't open a browser.

**Review by tailhook — APPROVED** [review]

File: src/commands/ui.rs (hunk 2 above)
Comment (fantix) [reply src/commands/ui.rs]: Fixed!

**Pull request merged**